integration/container/stats_test.go

```diff
@@ -0,0 +1,53 @@
+package container
+
+import (
+	"context"
+	"encoding/json"
+	"testing"
+	"time"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/network"
+	"github.com/docker/docker/integration/util/request"
+	"github.com/gotestyourself/gotestyourself/poll"
+	"github.com/gotestyourself/gotestyourself/skip"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAPIStatsContainerGetMemoryLimit(t *testing.T) {
```

Comment:
Maybe just call this `TestStats` so that if we decide we want to test more of the return value we can test it all in this one test, instead of creating new test cases for each field? They should all be independent so can probably be tested all together.

Reply:
Update. Thanks.